[2024-06-07T00:09:02.108+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-07T00:09:02.154+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_704449 scheduled__2024-06-06T00:00:00+00:00 [queued]>
[2024-06-07T00:09:02.167+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_704449 scheduled__2024-06-06T00:00:00+00:00 [queued]>
[2024-06-07T00:09:02.169+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-06-07T00:09:02.189+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_index_id_704449> on 2024-06-06 00:00:00+00:00
[2024-06-07T00:09:02.198+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4304) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-07T00:09:02.200+0000] {standard_task_runner.py:63} INFO - Started process 4305 to run task
[2024-06-07T00:09:02.198+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'load_data_mb', 'load_index_id_704449', 'scheduled__2024-06-06T00:00:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/load_data_mb.py', '--cfg-path', '/tmp/tmpg52w86a0']
[2024-06-07T00:09:02.202+0000] {standard_task_runner.py:91} INFO - Job 88: Subtask load_index_id_704449
[2024-06-07T00:09:02.287+0000] {task_command.py:426} INFO - Running <TaskInstance: load_data_mb.load_index_id_704449 scheduled__2024-06-06T00:00:00+00:00 [running]> on host e6f094d2943c
[2024-06-07T00:09:02.424+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_data_mb' AIRFLOW_CTX_TASK_ID='load_index_id_704449' AIRFLOW_CTX_EXECUTION_DATE='2024-06-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-06T00:00:00+00:00'
[2024-06-07T00:09:02.425+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-07T00:09:11.893+0000] {logging_mixin.py:188} INFO - {'status': 'success', 'info': [{'index_name': 'индекс реальных денежных доходов', 'index_id': 704449, 'dic_names': ['Регионы'], 'period_name': 'Год', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'индекс реальных денежных доходов', 'index_id': 704449, 'dic_names': ['Регионы'], 'period_name': 'Квартал', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'индекс реальных денежных доходов', 'index_id': 704449, 'dic_names': ['Регионы'], 'period_name': 'Месяц', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'индекс реальных денежных доходов', 'index_id': 704449, 'dic_names': ['Регионы'], 'period_name': 'Месяц с накоплением', 'info': 'ошибка талдау', 'error_code': 500}]}
[2024-06-07T00:09:11.894+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-07T00:09:11.895+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-07T00:09:11.909+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=load_data_mb, task_id=load_index_id_704449, execution_date=20240606T000000, start_date=20240607T000902, end_date=20240607T000911
[2024-06-07T00:09:11.975+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-07T00:09:11.993+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-07T00:09:11.999+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
